repo-issues/lib/template.html

```diff
@@ -0,0 +1,47 @@
+<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+<link rel="stylesheet" href="{{{__dirname}}}/style.css">
+
+<div class="container">
+    {{#repos}}
+    <div class="cell">
+        <div class="header">
+            <span class="title">
+                <img class="icon" 
+                    alt="{{language}}" 
+                    src="https://cdn.rawgit.com/konpa/devicon/master/icons/{{language}}/{{language}}-plain.svg"
+                    onerror="this.style.display='none'"/>
```

Comment:
Hides the image if no icon is found for a given language 👌

Reply:
nice touch!